src/app/items/services/items.service.ts

```diff
@@ -5,7 +5,9 @@ import { FirebaseService } from '../../services/firebase/firebase.service';
 import { CollectionType } from '../../services/firebase/models/collection-type.model';
 import { IFirebaseService } from '../../services/firebase/models/firebase-service.interface';
 
-@Injectable()
+@Injectable({
+  providedIn: 'root'
```

Comment:
remove providedIn root, and provide this service in the providers section in the relevant , module

- [ ] remove